screens/Settings/Revoke.tsx

```diff
@@ -118,16 +125,19 @@ export const Revoke: React.FC<RevokeScreenProps> = (props) => {
                 </Text>
                 <Text margin="0 0 12 0">
                   {t('revokingVids', {
-                    count: controller.selectedVidKeys.length,
+                    count: controller.selectedVidUniqueIds.length,
                   })}
                 </Text>
-                {controller.selectedVidKeys.map((vcKey, index) => (
+                {controller.selectedVidUniqueIds.map((uniqueId, index) => (
                   <View style={Theme.RevokeStyles.flexRow} key={index}>
                     <Text margin="0 8" weight="bold">
                       {'\u2022'}
                     </Text>
                     <Text margin="0 0 0 0" weight="bold">
-                      {vcKey.split(':')[2]}
```

Comment:
here we were rendering UIN. now are we are finding metadata but should also get what to render from that metadata